### DISCUSSION TITLE
Enabling node affinity for Kafka components

### DISCUSSION THREAD

**Discussion by talonx** [strimzi/strimzi-kafka-operator#5900] [Q&A] [answered]
I am unable to set node affinity for Kafka components like entity-operator, cruise control, etc when I deploy the Strimzi operator and some Kafka clusters. 

Note that this is not about the Kafka pods themselves - those I am able to pin to specific nodes by using pod affinities.

Can you point me to the relevant documentation?

**Comment by scholzj** [comment, marked as answer]
You configure it exactly the same way as you configured it for Kafka broker pods. Just in different part of the custom resource: https://strimzi.io/docs/operators/latest/full/using.html#type-PodTemplate-reference
